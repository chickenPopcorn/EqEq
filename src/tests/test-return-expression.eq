SomeContext = {
  a = {  
        0;
      }
  b = {
  		  3 + 4;
  	  }
  c = {
        3 - 1;
      }
  d = {
  	    3 % 2;
      }
  e = {
  	    | - 42|;
      }
  f = {
        3 == 1;
      }
  h = {
        3 < 4;
      }
  i = {
        2 ^ 2;
      }
  j = {
        (3 + 4);
      }
  k = {
        |- ((3 + 4) / 2)^2 * ( 18 - 7 ) |;
      }

}

SomeContext:find a {
  print("%.2f\n", a);
  print("%.2f\n", b);
  print("%.2f\n", c);
  print("%.2f\n", d);
  print("%.2f\n", e);
  print("%.2f\n", f);
  print("%.2f\n", h);
  print("%.2f\n", i);
  print("%.2f\n", j);
  print("%.2f\n", k);
}