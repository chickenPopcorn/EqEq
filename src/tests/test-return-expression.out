0.00
7.00
2.00
1.00
42.00
0.00
1.00
4.00
7.00
443.00
